src/celldega/nbhd/__init__.py

```diff
@@ -205,3 +213,190 @@ def alpha_shape_geojson(gdf_alpha, meta_cluster, inst_alpha):
+
+    with open(os.path.join(path_landscape_files, 'landscape_parameters.json'), 'r') as landscape_parameters_file:
+        landscape_parameters = json.load(landscape_parameters_file)
+
+    if landscape_parameters['technology'] == 'Xenium':
+        x = 'x_location'
+        y = 'y_location'
+
+    elif landscape_parameters['technology'] == 'MERSCOPE':
+        x = 'center_x'
+        y = 'center_y'
+
+    else:
+        x = 'x'
+        y = 'y'
+
+    print("Reading transcripts file...")
+    if isinstance(path_segmentation_files, str):
+        trx = pd.read_parquet(os.path.join(path_segmentation_files, "transcripts.parquet"))
+
+    else:
+        trx = pd.read_parquet(os.path.join(path_default_data, "transcripts.parquet"))
+
+    print("Transcripts file read.")
+
+    gdf_trx = gpd.GeoDataFrame(trx, geometry=gpd.points_from_xy(trx[x], trx[y]))
+    gdf_trx['geometry_image_space'] = gdf_trx['geometry'].apply(lambda geom: affine_transform(geom, [transformation_matrix[0, 0],
+                                                                                               transformation_matrix[0, 1],
+                                                                                               transformation_matrix[1, 0],
+                                                                                               transformation_matrix[1, 1],
+                                                                                               transformation_matrix[0, 2],
+                                                                                               transformation_matrix[1, 2]]))
+
+    gdf_trx.set_geometry('geometry_image_space', inplace=True)
+
+    print("Assignment of transcripts started...")
+
+    hexatile_assigned_trx = assigning_transcripts(gdf_polygons = gdf_hexatile,
+                                                  gdf_transcripts = gdf_trx)
+
+    gdf_hexatile_assigned_trx = gpd.GeoDataFrame(hexatile_assigned_trx,
+                                                      geometry=gpd.points_from_xy(hexatile_assigned_trx[x], hexatile_assigned_trx[y]))
+
+    gdf_hexatile_assigned_trx.set_geometry('geometry_image_space', inplace=True)
+
+    gdf_hexatile_assigned_trx.to_parquet(os.path.join(path_landscape_files, "hexatile_assigned_trx.parquet"))
+
+    print("Assignment of transcripts done and saved.")
+
+    print("Calculating percentage of hexatile-specific unassigned transcripts...")
+
+    unassigned_counts = gdf_hexatile_assigned_trx[gdf_hexatile_assigned_trx['cell_id'] == "UNASSIGNED"].groupby('polygon_index').size()
+    total_counts = gdf_hexatile_assigned_trx.groupby('polygon_index').size()
+
+    percentage_unassigned = (unassigned_counts / total_counts) * 100
+    percentage_unassigned = percentage_unassigned.fillna(0)
+
+    percentage_unassigned = percentage_unassigned[percentage_unassigned < 75]
```

Comment:
Instead of hardwiring here, can you create an argument for value 75 so users can change if needed? Thx.